packages/office-ui-fabric-react/src/components/DetailsList/__snapshots__/DetailsHeader.test.tsx.snap

```diff
@@ -1474,31 +1510,8 @@ exports[`DetailsHeader renders accessible labels 1`] = `
           
         </i>
       </span>
-    </span>
+    </div>
   </div>
-  <label
-    className=
-
-        {
-          border: 0px;
-          height: 1px;
-          margin-bottom: -1px;
-          margin-left: -1px;
-          margin-right: -1px;
-          margin-top: -1px;
-          overflow: hidden;
-          padding-bottom: 0px;
-          padding-left: 0px;
-          padding-right: 0px;
-          padding-top: 0px;
-          position: absolute;
-          width: 1px;
-        }
-    id="header4-b-tooltip"
-  >
-    Click to sort.
```

Comment:
I don't see this text emitted anymore. These strings still need to remain as the content of the element pointed to by `aria-describedby`.

Reply:
No longer applicable, thank you for the heads-up though.